src/ProjectTemplates/test/WorkerTemplateTest.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Test
+{
+    public class WorkerTemplateTest
+    {
+        public WorkerTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelper output)
+        {
+            ProjectFactory = projectFactory;
+            Output = output;
+        }
+
+        public Project Project { get; set; }
+        public ProjectFactoryFixture ProjectFactory { get; }
+        public ITestOutputHelper Output { get; }
+
+        [Fact(Skip = "Microsoft.NET.Sdk.Worker isn't available yet")]
+        public async Task WorkerTemplateAsync()
+        {
```

Comment:
Same comment as for the grpc template. Shouldn't we run this if it's an executable template?

/cc: @glennc